### PR TITLE
Add Drawer to _layout.svelte

### PR DESCRIPTION
**Pull request by hobbitronics**
- added drawer and fixed layout
- moved css rule to body
- arrange imports and remove extra {}
- made Drawer menu narrower - will work with ui-components update as no class prop now
- capitalize menu items

### CODE REVIEW

File: src/pages/_layout.svelte [hunk 1]
```diff
@@ -1,27 +1,52 @@
 <script>
-import AppFooter from '../components/AppFooter.svelte'
-import AppHeader from '../components/AppHeader.svelte'
-
+  import AppFooter from '../components/AppFooter.svelte'
```
Comment (forevermatt) [inline src/pages/_layout.svelte:2]: Completely optional, but it looks like something added a 2-space initial indent for everything in the `<script>` tag, which you could remove.

File: src/pages/_layout.svelte [hunk 2]
```diff
@@ -1,27 +1,52 @@
...
+      icon: 'label',
+      label: 'Claims',
+    },
+    {
+      url: '/faq',
+      icon: 'article',
+      label: 'FAQ',
+    },
+    {
+      url: '/chat',
```
Comment (forevermatt) [inline src/pages/_layout.svelte:25]: Do we want to leave this out until the chat feature is within scope? Either is fine, just asking.

**Review by forevermatt — APPROVED** [review]

File: src/pages/_layout.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/pages/_layout.svelte]: I will comment it out for now, if it will be a long time we can delete.

File: src/pages/_layout.svelte (hunk 2 above)
Comment (hobbitronics) [reply src/pages/_layout.svelte]: yeah, I will comment it out at least for now.

**Pull request merged**